.github/workflows/run-e2e-at-night.yml

```diff
@@ -0,0 +1,32 @@
+name: Cypress nightly tests
+on:
+  pull_request:
+    types: [reopened, synchronize, labeled, opened]
+    branches: ["*"]
```

Comment:
At the moment this would cause 30+ full automation tests to run every day, are we sure this is what we want? cc @patrys

Reply:
@jwm0 , @michalina-graczyk  As I mention in comment above I will remove this part, I added it just to show that tests are running on correct env